qutebrowser/mainwindow/prompt.py

```diff
@@ -633,6 +650,35 @@ def __init__(self, question, parent=None):
         self._to_complete = ''
         self._root_index = QModelIndex()
 
+        self._history = cmdhistory.History(parent=self)
+        fprompt_history = objreg.get('fprompt-history')
```

Comment:
I'd still prefer naming this "filename_prompt_history`, even in code. If you see this without context, it's totally unclear what "fprompt" is.